### PR TITLE
feat: show ticket amount details

### LINKED ISSUE #266: Show ticket details in attendee fragment
working on it
To be done after https://github.com/fossasia/open-event-android/pull/282

### PR DESCRIPTION
**Pull request by nikit19**
fixes #266 
Removed `if (it.toFloat() > 0)` from attendeeViewModel because for free orders the quantity was not calculated correctly.
Shifted nested scrollview to the top in attendee fragment
Hardcoded the currency to dollar for now as it is being worked upon in #316 
ScreenShot:
screenshot_20180713-155523

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/ticket/TicketDao.kt [hunk 1]
```diff
@@ -23,4 +23,7 @@ interface TicketsDao {
 
     @Query("SELECT price from Ticket WHERE id in (:ids)")
     fun getTicketPriceWithIds(ids : List<Int>): Single<List<String>>
+
+    @Query("SELECT * from Ticket WHERE id in (:ids)")
+    fun getTicketsWithIds(ids : List<Int>): Single<List<Ticket>>
```
Comment (dreadpool2) [inline app/src/main/java/org/fossasia/openevent/general/ticket/TicketDao.kt]: Remove space between `ids` and `:`

File: app/src/main/java/org/fossasia/openevent/general/ticket/TicketDao.kt [hunk 2]
```diff
@@ -23,4 +23,7 @@ interface TicketsDao {
 
     @Query("SELECT price from Ticket WHERE id in (:ids)")
     fun getTicketPriceWithIds(ids : List<Int>): Single<List<String>>
```
Comment (dreadpool2) [inline app/src/main/java/org/fossasia/openevent/general/ticket/TicketDao.kt]: same here

**Comment by nikit19** [comment]
PR updated

**Comment by iamareebjamal** [comment]
Umm, how about wait for at least 2 people to review and mentor to check?

**Pull request merged**